locales/translations.json

```diff
@@ -52,7 +52,7 @@
         "cvePairStatusModal.useOverallCve": "Use overall CVE status",
         "cveStatusModal.updateSuccessful": "Status updated for {cveCount} CVEs",
         "cveStatusModal.titlePlural": "Set status for these CVEs and all systems",
-        "cveStatusModal.titleSingle": "Set status for this CVEs and all systems",
+        "cveStatusModal.titleSingle": "Set status for this CVE and all systems",
```

Comment:
checkout how internationalization and pluralization are done [here](https://github.com/RedHatInsights/vulnerability-ui/pull/227/files#diff-ba88122e3a7f7255857b6f550455ea07R53)  and [here](https://github.com/RedHatInsights/vulnerability-ui/pull/227/files#diff-ba0f59f3a1f4d8189c63f7f3f45470b4R91) .
More examples by the library https://github.com/formatjs/react-intl/blob/master/examples/Advanced.tsx